default/specials.txt

```diff
@@ -1311,7 +1313,7 @@ Special
                 Ship
             ]
             effects = [
-                SetDetection value = Value + 60
+                SetDetection value = Min(Target.Construction / 30, 1) * Value + 60
```

Comment:
Pointless? Ships don't have a construction value and the special can only affect ships.